meeting-notes/2018/2018-09-10--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,144 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 September 10, 2018
+
+- **Lead:** Alan Shaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @alanshaw
+  - @achingbrain
+  - @travisperson
+  - @jacobheun
+  - @vasco-santos
+  - @vmx
+- **Recording:** https://youtu.be/KhsdFb_b8Yo
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@alanshaw
+- Done:
+    - Published release candidate for `js-ipfs` 0.32.0-rc.1 and 0.32.0-rc.2
+    - Test stabilisation work [#1541](https://github.com/ipfs/js-ipfs/pull/1541)
+    - Sent PR for faster startup time when using CLI with no daemon running [#1542](https://github.com/ipfs/js-ipfs/pull/1542)
+    - Many fixes to API docs [#1538](https://github.com/ipfs/js-ipfs/pull/1538)
+    - Made progress adding `--cid-base` option to _all_ relevant `js-ipfs` commands (working in branch [feat/cid-base-option](https://github.com/ipfs/js-ipfs/tree/feat/cid-base-option))
+- Blocked:
+    - Jenkins npm errors frequent [#51](https://github.com/ipfs/testing/issues/51)
+    - Jenkins windows workers out of space [#135](https://github.com/ipfs/testing/issues/135)
+- Next:
+    - Release js-ipfs 0.32
+    - Finish `--cid-base` option
+    - Create interop tests for block store get by CIDv0/1
+    - Implement block store get by CIDv0/1
+
+@achingbrain
+- Done:
+  - Spun up npm registry at https://registry.js.ipfs.io
+    - Play along at home with `npm i --registry=https://registry.js.ipfs.io`
```

Comment:
Wooot!! :D This is nice :D 

@achingbrain, Does it have the whole registry? What's the repo size?

Reply:
At the moment it's configured to only fetch individual tarballs when they've been requested by a client and aren't present locally as the EC2 instance it's running on doesn't have enough storage to eagerly fetch the whole thing.

This issue is to sort out a more scalable storage solution, once that's in place we can hold whole the whole repo: https://github.com/ipfs/infrastructure/issues/432

Reply:
Did you have the chance to download the entire registry locally to see the size it took?